test/functional/viml/completion_spec.lua

```diff
@@ -312,6 +312,51 @@ describe('completion', function()
       feed('i<C-r>=TestComplete()<CR><ESC>')
       eq(0, eval('&l:modified'))
     end)
+    it('does not ignores the generated candidates if iequal', function()
+      execute('set completeopt+=menuone,iequal')
+      execute('set ignorecase')
```

Comment:
`ignorecase` is used the default completion narrowing.
Note: It is not documented...
I will add `noignorecase` test.